src/connectors/index.ts

```diff
@@ -32,6 +32,10 @@ export const walletlink = new WalletLinkConnector({
   appLogoUrl: 'https://raw.githubusercontent.com/pangolindex/interface/master/public/images/384x384_App_Icon.png'
 })
 
-export const walletconnect = new WalletConnectConnector({ rpc: {
-    1: NETWORK_URL
-  }, qrcode: true, bridge: 'https://bridge.walletconnect.org'})
\ No newline at end of file
+export const walletconnect = new WalletConnectConnector({
```

Comment:
@AlexBHarley thanks for submitting this through. 

My only question here becomes whether we should be hardcoding the Network URL. Theoretically TrustWallet wouldn't work with Fuji anyway so I guess it's acceptable.